avionics/common/include/tasks/restart_camera.hpp

```diff
@@ -0,0 +1,24 @@
+#pragma once
+#include "cameras.h"
+#include <unistd.h>
```

Comment:
That's a UNIX header include; the sleep functionality should be provided by HAL.